pixel/head.html

```diff
@@ -3,14 +3,44 @@
 	var callback = function(){
 	  // Handler when the DOM is fully loaded
 
+	  	// add style tag in document head with custom color for button
+	  	if('{{settings.customColor}}' != ''){
+	  		const color = window.decodeURIComponent('{{settings.customColor}}');
+			const styles = `
+				.embeddedServiceHelpButton .helpButton .uiButton {
+					background-color: ${color} !important;
+				}
+				.embeddedServiceHelpButton .helpButton .uiButton:focus {
+					outline: 1px solid ${color} !important;
+				}
+			`;
+
+			const styleSheet = document.createElement("style");
+			styleSheet.type = "text/css";
+			styleSheet.innerText = styles;
+			document.head.appendChild(styleSheet);
```

Comment:
That's weird.. Doesnt the settings have a theme settings?

Reply:
Also, you shouldn't release a new version in a branch.

Reply:
Thanks Breno, I agree it's weird adding styles that way. I've just removed it, we'll see if that can be done in other ways later.
I didn't understand what you said about the new version release, I didn't change the version number in the manifest, it's still 0.0.3, not sure what you meant.

Reply:
I mean this commit here:
https://github.com/vtex-apps/salesforce-chat/pull/5/commits/eaa13631b5bc1d3df6b69135a55c536d55338943

Reply:
Oh that was @ccamargo2294 in the previous release.